frontend/common/src/components/form/Select/Select.tsx

```diff
@@ -93,15 +150,37 @@ const Select: React.FunctionComponent<SelectProps> = ({
               {nullSelection}
             </option>
           )}
-          {options.map((option) => (
-            <option
-              aria-label={option.ariaLabel}
-              key={option.value}
-              value={option.value}
-            >
-              {option.label}
-            </option>
-          ))}
+          {optionsModified.map((option) =>
+            Object.prototype.hasOwnProperty.call(option, "options") ? (
```

Comment:
See previous comment about the `in` operarator

Reply:
See https://github.com/GCTC-NTGC/gc-digital-talent/pull/5729#discussion_r1110012421